packages/kbn-plugin-helpers/README.md

```diff
@@ -29,17 +42,11 @@ $ plugin-helpers help
 
 ## Versions
 
-Plugin Helpers | Kibana
--------------- | ------
-9.x | 7.0+ (`kibanaRoot` setting removed from `plugin-helpers`)
-8.x | 7.0+
-7.x | 4.6.x to 6.x (node 6+ only)
-6.x | 4.6.x to 6.x
-5.x | 4.x
+The plugins helpers in the Kibana repo are available for Kibana 6.3 and greater. Just checkout the branch of Kibana you want to build against and the plugin helpers should be up to date for that version of Kibana.
```

Comment:
Should we also link to `@elastic/plugin-helpers` for <=6.2?

Reply:
I suppose